### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by brhanuhailu**
### Portfolio - mobile menu and Milestone 👍 
- [X] Added mobile menu using JavaScript and Just selected template three
- [X] Checked Linter error
- [X] Used JavaScript events.
- [X] Configure Eslin for JavaScript checker

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the javascript linters also setup, for this project both the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be configured
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by brhanuhailu** [comment]
Dear @prolajumokeoni we are little bit not understanding  the comment. we have checked the JavaScript and linter error checker and all checked have passed. can we have some detail comment/information  please?

**Comment by prolajumokeoni** [comment]
Hi Team, you are missing  the  Eslint check, please add the config  in your linters.yml file, for this project both the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be configured

Screenshot_685

**Comment by brhanuhailu** [comment]
Dear @prolajumokeoni Thank you very much for your constructive comment

File: index.html [hunk 1]
```diff
@@ -12,14 +12,26 @@
 <body>
     <header class="header-container">
         <div class="logo">Welcome</div>
-        <button type="button"><i class="fa fa-bars" class="humberger"></i></button>
+        <button type="button" id="menu-bar"><i class="fa fa-bars"></i></button>
         <nav>
             <ul class="header-nav-desktop">
                 <li><a href="#">Portfolio</a></li>
```
Comment (osamaashraf6) [inline index.html:18]: - [ ] Great implementing 😃 the js functionality for the mobile menu, but I see that there is a problem with the style I see that the hamburger icon not aligning like the Figma design at the correct position, so please try to add the hamburger at the correct position and with that your portfolio will be attractive for the visitors 😊
- [ ] Also consider adding some padding to the header to be like the Figma design 💯

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   image |   image |

File: index.html [hunk 2]
```diff
@@ -12,14 +12,26 @@
...
                 <li><a href="#">About</a></li>
                 <li><a href="#">Contract</a></li>
             </ul>
         </nav>
+        <div class="mobile-menu" id="menu-mobile">
+            <div class="menu-header">
+                <span class="close">&times;</span>
```
Comment (osamaashraf6) [inline index.html:25]: - [ ] You did well 😃at adding the time's icon, but I see that the margin between the icon and the ul navs is bigger than the Figma Design , so please try to decrease the margin
- [ ]  Great job ,but I see that the margin of the navs here is bigger than the Figma Design, so please try to decrease the margin between the navs to be like the Figma Design 😊


> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   image |  image |

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @brhanuhailu 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights ✔️

- [x] Descriptive READme file in a professional way.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow
- [x] Beautiful UI. 👌
- [x] Nice design.




## Required Changes ♻️

- [ ]  check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Zuhaib042 — APPROVED** [review]
Hi @brhanuhailu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve
✅ No linters errors
✅ Professional PR
## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by brhanuhailu** [comment]
@Zuhaib042  Thank you very much

**Pull request merged**